CODEOWNERS

```diff
@@ -274,9 +274,9 @@ geode-core/**/org/apache/geode/internal/cache/control/**          @kirklund @Don
 # Testing utilities
 #-----------------------------------------------------------------
 #geode-concurrency-test/**
-geode-dunit/**                                                    @mhansonp @kirklund
-geode-junit/**                                                    @mhansonp @kirklund
-geode-jmh/**                                                      @pivotal-jbarrett @kirklund
```

Comment:
did you mean to remove Kirk from jmh?

Reply:
No Kirk apparently did that. I had a merge conflict.